bin/test-kube-e2e

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+GIT_ROOT="${GIT_ROOT:-$(git rev-parse --show-toplevel)}"
+. "${GIT_ROOT}/bin/include/versioning"
+
+export DOCKER_IMAGE_TAG=${VERSION_TAG}
+export TEST_NAMESPACE="default"
```

Comment:
Could we use "test$(date +%s)" namespace to do e2e as integration-test does? If tests are triggered in parallel, we should keep tests didn't influence each other.

Reply:
Hi, this is intended to run on another cluster as part of the CI. If I run this locally, I usually do all tests sequentially. Do you have a use case where you run locally everything in parallel? 
I think I did not test this running in another namespace besides default.

Reply:
@edwardstudy if you consider that this test should use a custom ns, please request it as a change, so I can include it in this PR :)

Reply:
I understood that e2e tests will be run in CI and we need to trigger CI builds in parallel.

Reply:
@qu1queee I think we should support a test namespace and helm installing the operator into it. Not sure if that's in the scope of your story? Anyhow let's do that in a separate PR

Reply:
its not, but I agree we should. I prefer to have it in another PR.

Reply:
@edwardstudy you were right. I was not aware we run multiple PRs check in parallel. In any case I already add the test ns support.